src/components/experiences/positive-experiences.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
```

Comment:
this and negative-experiences should just be one generalized component with configs